### ISSUE TITLE
dapple run: could not unlock signer account

### ISSUE THREAD

**Issue by jorisbontje** [nexusdev/dapple#218]
The following error happens with commit 4b93686f561ae4ec1128da937b50d044aaa67e96 and seems to be introduced in the recently added `configurableDefaultAccount` feature #207 

```
$ dapple run deploy.ds
Using local solc installation...
Deploying to environment "default"...
Connected to RPC client, block height is 3
simulated successfully

.
  could not unlock signer account
```

This error happens both with and without the new `account` setting in `~/.dapplerc`. Also the account IS unlocked in geth, as verified via `web3.eth.sign` in `geth attach`

With 1 commit back (95e7ecc6dd4dad76ec4eeb1216e1c0c3986fc510) this error doesn't happen:
```
$ dapple run deploy.ds
Using local solc installation...
Deploying to environment "default"...
Connected to RPC client, block height is 3
simulated successfully
DEPLOYED: AtomicTrade = 0x20cea38fb560e03d325d12d0f0feba05811d69ba
GAS COST: 447410 for "new AtomicTrade"
CONFIRMED deploy after 1 blocks!

.
  Successfully deployed!
```

**Comment by jorisbontje** [comment]
The problem persists; with commit bf905badad4e671521b7c4fde984080ab891068b and same way to reproduce as described above.

**Comment by mhhf** [comment]
@jorisbontje can you check out the newest dapple from master and close this issue if this is resolved for you?!

**Comment by jorisbontje** [comment]
yes, working fine now!

**Comment by raineorshine** [comment]
I am getting this error on Dapple v0.7.1, TestRPC v2.0.5:

```
$ dapple run foo.dapplescript -e live
/bin/sh: solc: command not found
No local solc found. Failing over to JS compiler...
simulated successfully
Deploying to environment "live"...
Connected to RPC client, block height is 10
/Users/raine/.nvm/versions/node/v5.5.0/lib/node_modules/dapple/node_modules/solc/bin/soljson-latest.js:1
(function (exports, require, module, __filename, __dirname) { var Module;if(!Module)Module=(typeof Module!=="undefined"?Module:null)||{};var moduleOverrides={};for(var key in Module){if(Module.hasOwnProperty(key)){moduleOverrides[key]=Module[key]}}var ENVIRONMENT_IS_WEB=typeof window==="object";var ENVIRONMENT_IS_WORKER=typeof importScripts==="function";var ENVIRONMENT_IS_NODE=typeof process==="object"&&typeof require==="function"&&!ENVIRONMENT_IS_WEB&&!ENVIRONMENT_IS_WORKER;var ENVIRONMENT_IS_SHELL=!ENVIRONMENT_IS_WEB&&!ENVIRONMENT_IS_NODE&&!ENVIRONMENT_IS_WORKER;if(ENVIRONMENT_IS_NODE){if(!Module["print"])Module["print"]=function print(x){process["stdout"].write(x+"\n")};if(!Module["printErr"])Module["printErr"]=function printErr(x){process["stderr"].write(x+"\n")};var nodeFS=require("fs");var nodePath=require("path");Module["read"]=function read(filename,binary){filename=nodePath["normalize"](filename

Error: Error: could not unlock signer account
    at Blockchain.queueAction (/Users/raine/.nvm/versions/node/v5.5.0/lib/node_modules/ethereumjs-testrpc/lib/blockchain.js:384:21)
    at Blockchain.queueTransaction (/Users/raine/.nvm/versions/node/v5.5.0/lib/node_modules/ethereumjs-testrpc/lib/blockchain.js:339:8)
    at Manager.eth_sendTransaction (/Users/raine/.nvm/versions/node/v5.5.0/lib/node_modules/ethereumjs-testrpc/lib/manager.js:157:19)
    at Manager.handleRequest (/Users/raine/.nvm/versions/node/v5.5.0/lib/node_modules/ethereumjs-testrpc/lib/manager.js:75:10)
    at next (/Users/raine/.nvm/versions/node/v5.5.0/lib/node_modules/ethereumjs-testrpc/node_modules/web3-provider-engine/index.js:93:18)
    at VmSubprovider.handleRequest (/Users/raine/.nvm/versions/node/v5.5.0/lib/node_modules/ethereumjs-testrpc/node_modules/web3-provider-engine/subproviders/vm.js:42:12)
    at next (/Users/raine/.nvm/versions/node/v5.5.0/lib/node_modules/ethereumjs-testrpc/node_modules/web3-provider-engine/index.js:93:18)
    at GethDefaults.handleRequest (/Users/raine/.nvm/versions/node/v5.5.0/lib/node_modules/ethereumjs-testrpc/lib/gethdefaults.js:17:12)
    at next (/Users/raine/.nvm/versions/node/v5.5.0/lib/node_modules/ethereumjs-testrpc/node_modules/web3-provider-engine/index.js:93:18)
    at FilterSubprovider.handleRequest (/Users/raine/.nvm/versions/node/v5.5.0/lib/node_modules/ethereumjs-testrpc/node_modules/web3-provider-engine/subproviders/filters.js:76:7)
    at Object.module.exports.InvalidResponse (/Users/raine/.nvm/versions/node/v5.5.0/lib/node_modules/dapple/node_modules/web3/lib/web3/errors.js:35:16)
    at /Users/raine/.nvm/versions/node/v5.5.0/lib/node_modules/dapple/node_modules/web3/lib/web3/requestmanager.js:86:36
    at request.onreadystatechange (/Users/raine/.nvm/versions/node/v5.5.0/lib/node_modules/dapple/node_modules/web3/lib/web3/httpprovider.js:114:13)
    at dispatchEvent (/Users/raine/.nvm/versions/node/v5.5.0/lib/node_modules/dapple/node_modules/xmlhttprequest/lib/XMLHttpRequest.js:591:25)
    at setState (/Users/raine/.nvm/versions/node/v5.5.0/lib/node_modules/dapple/node_modules/xmlhttprequest/lib/XMLHttpRequest.js:610:14)
    at IncomingMessage.<anonymous> (/Users/raine/.nvm/versions/node/v5.5.0/lib/node_modules/dapple/node_modules/xmlhttprequest/lib/XMLHttpRequest.js:447:13)
    at emitNone (events.js:85:20)
    at IncomingMessage.emit (events.js:179:7)
    at endReadableNT (_stream_readable.js:906:12)
    at nextTickCallbackWith2Args (node.js:474:9)
```

foo.dapplescript:
```
var box = new Box.value(1000000)("Box")
box.Set.value(100000)(42)
```

~/.dapplerc:
```
environments:
  default:
    ethereum: internal
  evm: default
  live:
    ethereum:
      host: localhost
      port: '8545'
      account: '0x825c74b1fbec73911379b30e1b0ba89ff40e662a'
    ipfs:
      host: localhost
      port: '5001'
```

**Comment by mhhf** [comment]
This should be no issue with dapple 0.8, since default account is now a required property for an environment. As for working with testrpc - testrpc changes the accounts with each start, if testrpc is restarted the saved default Account is removed and hence unlocked.
Closing, since resolved with 0.8.
